test/twitch-proxy.test.js

```diff
@@ -0,0 +1,37 @@
+// TODO: Remove once types for Jest are recognized
+/* eslint-disable no-undef */
+const axios = require('axios');
+const portMap = require('../port-map.json');
+const twitchProxyPort = portMap['twitch-proxy'];
+
+describe('kraken api', () => {
```

Comment:
I feel silly for asking, but why/what is kraken relating to?

Reply:
It refers to the version of Twitch API FreeCodeCamp is mocking. 

This file holds all of the [Kraken](https://github.com/freeCodeCamp/demo-projects/blob/main/apps/twitch-proxy/static-data/kraken.json) dataset.